canonical-web-and-design/www.ubuntu.com
# Add blog module and homepage

## Done

- Adds blog module
- Adds homepage template
- Adds featured posts

## QA

- check `https://www-ubuntu-com-canonical-web-and-design-pr-5071.run.demo.haus/blog` on the demo service. Make sure the homepage blog posts, and featured posts look correct


## Issue / Card

Fixes https://github.com/canonical-web-and-design/base-squad/issues/428
Fixes https://github.com/canonical-web-and-design/base-squad/issues/429

## Screenshots
